/**:
  ros__parameters:
    border_x: [-2.7432,2.7432,2.7432,-2.7432]
    border_z: [-7.3152,-7.3152,6.096,6.096]